SPECS/usrsctp/usrsctp.signature.json

```
@@ -0,0 +1,5 @@
+{
+ "Signatures": {
+  "0.9.5.0.tar.gz": "260107caf318650a57a8caa593550e39bca6943e93f970c80d6c17e59d62cd92"
```

Please re-name the file to something easier to associate with the package. For instance:
```suggestion
  "usrsctp-0.9.5.0.tar.gz": "260107caf318650a57a8caa593550e39bca6943e93f970c80d6c17e59d62cd92"
```

---

Acknowledged. updated in the recent commit.